### PR TITLE
[WIP] Migrate doc content to new format

### PR DESCRIPTION
**Pull request by ashtul**
Adjust `commands.json` and each command `.md` file to the new format

### CODE REVIEW

File: docs/commands.yaml [hunk 1]
```diff
@@ -1,56 +0,0 @@
-FT.CREATE:
```
Comment (oshadmi) [inline docs/commands.yaml:1]: Is this still relevant? Not touched for 5 years - seems deprecated

**Review by oshadmi — COMMENTED** [review]
Small comments

File: docs/docs/administration/Upgrade_to_2.0.md [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
+v2 of RediSearch reachitects the way indices are kept in sync with the data. Instead of using `FT.ADD` command to index documents, RediSearch 2.0 follows hashes that match the index description regardless of how those were inserted or changed on Redis (`HSET`, `HINCR`, `HDEL`). The index description will filter hashes on a prefix of the key, and allows you to construct fine-grained filters with the `FILTER` option. This description can be defined during index creation (`FT.CREATE`). 
 
-v1.x indices (further referred to as legacy indices) don't have such index description. That is why you will need to supply their descriptions when upgrading to v2. During the upgrade to v2, you should add the descriptions via the module's configuration so RediSearch 2.0 will be able to load these legacy indexes.
+v1.x indices (further referred to as legacy indicegit s) don't have such index description. That is why you will need to supply their descriptions when upgrading to v2. During the upgrade to v2, you should add the descriptions via the module's configuration so RediSearch 2.0 will be able to load these legacy indexes.
```
Comment (rafie) [inline docs/docs/administration/Upgrade_to_2.0.md:18]: typo

**Review by oshadmi — APPROVED** [review]

File: .github/workflows/trigger-build.yml [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
 on:
+  workflow_dispatch:
   push:
     branches:
-      - master-newdocs
+      - master
+      - '[0-9]+.[0-9]+'
+    paths:
+      - 'mkdocs.yml'
+      - 'docs/**'
```
Comment (oshadmi) [inline .github/workflows/trigger-build.yml:11]: @ashtul 
Seems we should also include the path `commands.json`?

**Review by oshadmi — APPROVED** [review]

**Pull request merged**